src/LiveComponent/doc/index.rst

```diff
@@ -928,6 +928,8 @@ The following hooks are available (along with the arguments that are passed):
 * ``loading.state:finished`` args ``(element: HTMLElement)``
 * ``model:set`` args ``(model: string, value: any, component: Component)``
 
+.. _loading:
```

Comment:
Not a RST syntax expert here, but it's not missing something?

Reply:
The syntax is a bit weird, but this is basically used for referencing anchors